src/test/java/com/teragrep/pth_06/planner/walker/conditions/DocumentExtension.java

```diff
@@ -0,0 +1,26 @@
+package com.teragrep.pth_06.planner.walker.conditions;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolver;
+import org.w3c.dom.Document;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+
+/**
+ * Provides Document object for tests
+ */
+public final class DocumentExtension implements ParameterResolver {
```

Comment:
please add description why this is required.

Reply:
Deleted class after refactor